README.md

```diff
@@ -83,14 +91,16 @@ management studio, it should looks like this:
 
 On first look, this looks fine, but I want you to fix a couple of things / bugs:
 
+> **Note**: All codechanges you will do will be within the **Database** project(s). 
+
 * One of the foreign key fields in Claims has a name not adhering to standard (CoverEntityId):
 
     ![Wrong field names](/Images/Wrong_FK_Name.png)
 
     Hint! The ClaimsEntity lacks the (navigation) property which represents the FK. You can also decorate that field with an annotation `[ForeignKey("CoverId")]` to make the Entity class easier to read (or if you have schemas which does not allow the ef core engine to naturally resolve these FK references).
 
 * When running the initial migration above, there were some warnings written to the console. Get rid of them:
-    * The schemas now uses nvarchar(max) as type for the string fields. This has a performance penalty.
+    * The schemas has nvarchar(max) as type for the string fields. This has a performance penalty.
```

Comment:
This is not shown as a build warning or when scaffolding a migration. For me it only shows as a squigly on the fields.